docs/file-reference.md

```diff
@@ -134,14 +134,14 @@ Each "app" (Kedge file) is a Kubernetes <a target="_blank" href="https://kuberne
 | name | string   | yes          | The name of the app or micro-service this particular file defines. |
 | controller | string   | no           | The Kubernetes controller of the app or micro-service this particular file (default: "deployment") |
 | labels | object   | no           | Map of string keys and values that can be used to organize and categorize (scope and select) objects. May match selectors of replication controllers and services. |
-| containers | array of [containerSpec](#containerspec) | yes          | [containerSpec](#containerspec)  object |
-| volumeClaims | array of [persistentVolume](#persistentvolume) | no           | [persistentVolume](#persistentvolume) object |
-| configMaps | array of [configMap](#configmap) | no           | [configMap](#configmap) object |
-| services | array of [service](#service) | no           | [service](#service) object |
-| ingresses | array of [ingress object](#ingressobject) | no           | [ingress object](#ingressobject) object |
-| routes | array of [route object](#routeobject) | no           | [route object](#routeobject)  object |
-| secrets | array of [secret](#secret) | no           | [secret](#secret) object |
-| includeResources | array of [includeResources](#includeResources) | no           | 
+| containers | array of [containerObject](#containerobject) | yes          | [containerObject](#containerObject) |
```

Comment:
the first reference`(#containerobject)` is working but the second one `(#containerObject)` is not.
Anchors are always lowercase (at least for GitHub markdown)